__tests__/app.test.js

```diff
@@ -451,4 +451,15 @@ describe('12. DELETE /api/comments/:comment_id', () => {
       expect(body.msg).toBe('bad request')
     })
   })
+})
+
+describe('13. GET /api' , () => {
+  test('200 - responds with JSON object of endpoint descriptions', () => {
+    return request(app)
+    .get('/api')
+    .expect(200)
+    .then(({body}) => {
+      expect(typeof body.endpoints).toBe('object')
+    })
+  });
```

Comment:
We need to be a bit more thorough with the testing.  Check if the object we are receiving back has all the keys we are expecting it to.